sysdig/resource_sysdig_secure_policy.go

```diff
@@ -176,7 +176,14 @@ func policyToResourceData(policy *secure.Policy, d *schema.ResourceData) {
 			}}
 		}
 	}
-	_ = d.Set("actions", actions)
+
+	currentContainerAction := d.Get("actions.0.container").(string)
+	currentCaptureAction := d.Get("actions.0.capture").([]interface{})
+	// If the policy retrieved from service has no actions and the current state is default values,
+	// then do not set the "actions" key as it may cause terraform to think there has been a state change
+	if len(policy.Actions) > 0 || currentContainerAction != "" || len(currentCaptureAction) > 0 {
```

Comment:
Would this also work when policy has other action types - pause, stop, prevent drift, kill associated with it?

Reply:
Yes, the only time `len(policy.Actions)` is 0 is when there is no action associated with it.  Likewise, if it has one of the other action types, `currentContainerAction` will not be ""